tachyon/math/polynomials/univariate/univariate_evaluation_domain.h

```diff
@@ -214,6 +218,7 @@ class UnivariateEvaluationDomain : public EvaluationDomain<F, MaxDegree> {
   constexpr std::vector<F> EvaluatePartialLagrangeCoefficients(
       const F& tau, base::Range<T, true, IsEndInclusive> range) const {
     size_t size = range.GetSize();
+    // TODO(ashjeong): prevent the use of |CHECK()| in constexpr functions
```

Comment:
Please remove this, we can't use `CHECK()` when it's compiled with cuda.

Reply:
Do you mean to remove the following `CHECK()` line and the current added comment?